open-machine-learning-jupyter-book/prerequisites/python-programming-introduction.md

```diff
@@ -303,30 +407,48 @@ The Python interactive shell was printing without using `print` but on Visual St
 
 The file name is `helloworld.py`.
 
-```{code-cell}
-print(2 + 3)             # addition(+)
-print(3 - 1)             # subtraction(-)
-print(2 * 3)             # multiplication(*)
-print(3 / 2)             # division(/)
-print(3 ** 2)            # exponential(**)
-print(3 % 2)             # modulus(%)
-print(3 // 2)            # Floor division operator(//)
-
-# Checking data types
-print(type(10))          # Int
-print(type(3.14))        # Float
-print(type(1 + 3j))      # Complex number
-print(type('Asabeneh'))  # String
-print(type([1, 2, 3]))   # List
-print(type({'name':'Asabeneh'})) # Dictionary
-print(type({9.8, 3.14, 2.7}))    # Set
-print(type((9.8, 3.14, 2.7)))    # Tuple
-```
-
 To run the Python file check the image below. You can run the Python file either by running the green button on Visual Studio Code or by typing `Python helloworld.py` in the terminal.
 
 ![Running Python script](../../images/running_python_script.png)
 
+### Example code
+
+Now here is a example code for you, you can try to think about what the output of the following code is.
+
+Just have a try! 
+
+You can view the answers in the Python Tutor below, but be sure to come back to the answers after thinking about them.
+
+```
+x = [1, 2, 3]
+y = [4, 5, 6]
+z = y
+y = x
+x = z
+
+x = [1, 2, 3] # a different [1, 2, 3] list!
+y = x
+x.append(4)
+y.append(5)
+z = [1, 2, 3, 4, 5] # a different list!
+x.append(6)
+y.append(7)
+y = "hello"
+
+
+def foo(lst):
+    lst.append("hello")
+    bar(lst)
+
+def bar(myLst):
+    print(myLst)
+
+foo(x)
+foo(z)
+```
+
+<iframe width="800" height="500" frameborder="0" src="https://pythontutor.com/visualize.html#code=x%20%3D%20%5B1,%202,%203%5D%0Ay%20%3D%20%5B4,%205,%206%5D%0Az%20%3D%20y%0Ay%20%3D%20x%0Ax%20%3D%20z%0A%0Ax%20%3D%20%5B1,%202,%203%5D%20%23%20a%20different%20%5B1,%202,%203%5D%20list!%0Ay%20%3D%20x%0Ax.append%284%29%0Ay.append%285%29%0Az%20%3D%20%5B1,%202,%203,%204,%205%5D%20%23%20a%20different%20list!%0Ax.append%286%29%0Ay.append%287%29%0Ay%20%3D%20%22hello%22%0A%0A%0Adef%20foo%28lst%29%3A%0A%20%20%20%20lst.append%28%22hello%22%29%0A%20%20%20%20bar%28lst%29%0A%0Adef%20bar%28myLst%29%3A%0A%20%20%20%20print%28myLst%29%0A%0Afoo%28x%29%0Afoo%28z%29&cumulative=false&curInstr=0&heapPrimitives=nevernest&mode=display&origin=opt-frontend.js&py=3&rawInputLstJSON=%5B%5D&textReferences=false"> </iframe>
```

Comment:
Actually, you can just use this content to replace the deleted {code-cell} of Python file section.

Reply:
So, it means to don't want this board，but to replace the code in Python file. So do I use code cells or do I just show the code and let the user do it themselves?

Reply:
I mean you could just replace the content of existing code cell with your new code. In another word, please you don't need to create a new section, but just put your new content into that existing section 'Python file'.